.evergreen/scripts/integration-tests.sh

```diff
@@ -7,11 +7,9 @@
 # MONGODB_VERSION: latest, 4.2, 4.0
 # TOPOLOGY: server, replica_set, sharded_cluster
 # AUTH: auth, noauth
-# AUTHSOURCE
 # IPV4_ONLY: off, on
 # SSL: openssl, darwinssl, winssl, nossl
-# ORCHESTRATION_FILE: <file name in orchestration_configs/ without .json>
-#   If this is not set, the file name is constructed from other options.
+# ORCHESTRATION_FILE: <file name in DET configs/${TOPOLOGY}s/>
 # OCSP: off, on
```

Comment:
Suggest removing this comment block. The OCSP variable is no-longer-used. `run-orchestration.sh` [documents supported environment variables](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/a69349171a454a12d9c584d55ace8f00cef49ab6/.evergreen/run-orchestration.sh#L5-L19).

Reply:
Done.